cartoframes/data/observatory/catalog/repository/repo_client.py

```diff
@@ -1,6 +1,11 @@
-from ..subscriptions import get_subscription_ids
-from .....auth import Credentials
-from .....io.managers.context_manager import ContextManager
+import os
+import requests
+from urllib.parse import urlsplit, urlunsplit, SplitResult
+from .....auth import Credentials, get_default_credentials
+
+DEFAULT_USER = 'do-metadata'
+API_BASE_PATH = 'api/v4/data/observatory'
```

Comment:
okay, this is the client to move

Reply:
https://github.com/CartoDB/cartoframes/pull/1576